app/views/mailing_list/steps/_name.html.erb

```diff
@@ -1,27 +1,14 @@
-<h1>Get personalised guidance on teacher training in your inbox</h1>
+<h1>Free personalised teacher training guidance</h1>
 
 <p>
-If you're interested in training to teach in primary, secondary and special schools in England, get everything you need to know to start a career in teaching.
+Discover everything you need to know to start a teaching career in England, sent straight to your inbox. Personalised to you, we'll send you the latest information as well as advice and support.
```

Comment:
We say `personalised` in the h1 and in the text here - little bit repetitive or reinforcing the message?

Reply:
I think it reinforces the message that it's not just a generic mailing list